src/storekit.cs

```diff
@@ -205,7 +218,7 @@ interface SKPaymentQueue {
 		[Export ("cancelDownloads:")]
 		void CancelDownloads (SKDownload [] downloads);
 
-		[Mac (10, 15), iOS (13, 0)]
+		[Mac (10, 15), iOS (13, 0)] // NoWatch??
```

Comment:
Should this explicitly be labeled as `NoWatch`?

Reply:
yes (unless the headers says otherwise)

It often happens Apple will put macros for `watchOS(unavailable)` on an API even if the framework is not available. We normally don't add those (since it only increase the metadata size without any benefits)

Reply:
yes, [NoWatch] should be used.